transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.mapping.edit/generated/org/polarsys/kitalpha/transposer/rules/handler/rules/common/provider/MappingElementItemProvider.java

```diff
@@ -352,24 +352,27 @@ public void notifyChanged(Notification notification) {
 	protected void collectNewChildDescriptors(Collection<Object> newChildDescriptors, Object object) {
 		super.collectNewChildDescriptors(newChildDescriptors, object);
 
-				newChildDescriptors.add
-					(createChildParameter
-						(CommonPackage.Literals.MAPPING_ELEMENT__OWNED_DEFAULT_POSSIBILITY,
-						 CommonFactory.eINSTANCE.createMappingPossibility()));
 
+		newChildDescriptors.add
+			(createChildParameter
+				(CommonPackage.Literals.MAPPING_ELEMENT__OWNED_DEFAULT_POSSIBILITY,
```

Comment:
This is not really a modification, only a reformatting. Please undo.

Reply:
This is due to EMF generator !